file.txt

```diff
@@ -1 +1 @@
-world
+hello world
```

Comment:
This is how it was supposed to be at the first place. But it would be better if you capitalize 'H'.